datahub-web-react/src/app/ingestV2/ManageIngestionPage.tsx

```diff
@@ -18,6 +18,26 @@ import {
 import { useAppConfig } from '@app/useAppConfig';
 import { useShowNavBarRedesign } from '@app/useShowNavBarRedesign';
 
+interface Capability {
+    capability: string;
+    description: string;
+    supported: boolean;
+}
+
+interface PluginDetails {
+    capabilities: Capability[];
+    classname: string;
+    platform_id: string;
+    platform_name: string;
+    support_status: string;
+}
+
+interface CapabilitySummary {
+    generated_at: string;
+    generated_by: string;
+    plugin_details: Record<string, PluginDetails>;
+}
```

Comment:
can we add a comment that this type is based off of the type in python and vice-versa in the python file? this way if we make a change in one place we know where to make the corresponding change.